scripts/monitoring/cron-send-docker-metrics.py

```diff
@@ -22,9 +23,11 @@
     zs = ZaggSender()
     try:
         cli = AutoVersionClient(base_url='unix://var/run/docker.sock')
+        cli_storage = Client(base_url='unix://var/run/docker.sock')
         du = DockerUtil(cli)
+        Deadcontainers = cli_storage.containers(quiet=True,filters={'status':'exited'}) #get all the exited container
         du_dds = du.get_disk_usage()
-
+        print "data_space:",du_dds.data_space_percent_available
         keys = {
             'docker.storage.data.space.used': du_dds.data_space_used,
             'docker.storage.data.space.available': du_dds.data_space_available,
```

Comment:
Let's add a new key for tracking the number of exited containers. I think that would be something good to have a count of, especially since you're looking to use it for autohealing).